arm-sql/2014-04-01/swagger/capabilities.json

```diff
@@ -0,0 +1,332 @@
+                "name": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The edition name."
+                },
+                "status": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The status of the server edition.",
+                    "enum": [
+                        "Visible",
+                        "Available",
+                        "Default"
+                    ],
+                    "x-ms-enum": {
+                        "name": "CapabilityStatus"
+                    }
+                },
+                "supportedServiceLevelObjectives": {
+                    "readOnly": true,
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/ServiceObjectiveCapability"
+                    },
+                    "description": "The list of supported Service Objectives for the edition."
+                }
+            },
+            "description": "Represents the server edition capabilities."
+        },
+        "ServerVersionCapability": {
+            "properties": {
+                "name": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The server version name."
+                },
+                "status": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The status of the server version.",
+                    "enum": [
+                        "Visible",
+                        "Available",
+                        "Default"
+                    ],
+                    "x-ms-enum": {
+                        "name": "CapabilityStatus"
+                    }
+                },
+                "supportedEditions": {
+                    "readOnly": true,
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/EditionCapability"
+                    },
+                    "description": "The list of supported server editions."
+                }
+            },
+            "description": "Represents the server capabilities."
+        },
+        "LocationCapabilities": {
+            "properties": {
+                "name": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The region name."
+                },
+                "status": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The status for the region with respect to Azure SQL.",
+                    "enum": [
+                        "Visible",
+                        "Available",
+                        "Default"
+                    ],
+                    "x-ms-enum": {
+                        "name": "CapabilityStatus"
```

Comment:
also, please extract the data type you implicitly define here:
```
type: string
enum: ...
x-ms-enum: ...
```
into a top-level definition (with some generic description of what the enum represents) and just `$ref`erence it here and in all other places (valuable deduplication, especially in case of a change to the enum!)